osf/models/schema_response.py

```diff
@@ -228,14 +227,13 @@ def update_responses(self, updated_responses):
         if updated_responses:
             raise ValueError(
                 'Encountered unexpected keys while trying to update responses for '
-                f'SchemaResponse with id [{self.id}]: {", ".join(updated_responses.keys())}'
```

Comment:
Sending people the PK is not useful

Reply:
can you find and replace all instances of `self.id` in error messages with `self._id`?

Reply:
or else omit this change and I'll get it